airbyte-webapp/src/core/resources/Connection.ts

```diff
@@ -48,6 +53,9 @@ export default class ConnectionResource
   readonly destinationId: string = "";
   readonly status: string = "";
   readonly message: string = "";
+  readonly namespaceFormat: string = "";
```

Comment:
Is this where to define the default value for the namespace format?

Can we have it filled with `"${SOURCE_NAMESPACE}"` when the "customformat" enum is selected for the first time? 

This would give hints to the user that there is a special variable that can be resolved and replaced by the actual namespace from the source

Reply:
No. It is not where we define default values. 


How are we going to detect whether it's his first time when he selects source or not? What if user created connection with one value and then on update changed to source? We will not know that it's his first time. 

If necessary, we could set it as a default value but it will be always set by default when user switches enums. 

Also I suppose it's better to reflect such API feature in UI explicitly by adding some sort of components or controls instead of making them remember what values they need to set to allow one or another behavior

Reply:
> If necessary, we could set it as a default value but it will be always set by default when user switches enums.

That's fine

> Also I suppose it's better to reflect such API feature in UI explicitly by adding some sort of components or controls instead of making them remember what values they need to set to allow one or another behavior

Yes, since we have only one variable for the moment, we can include that in the description of the field?